TWLight/users/forms.py

```diff
@@ -126,7 +126,7 @@ def __init__(self, *args, **kwargs):
         self.helper.layout = Layout(
             "terms_of_use",
             # Translators: this 'I accept' is referenced in the terms of use and should be translated the same way both places.
-            Submit("submit", _("I accept"), css_class="btn btn-default"),
+            Submit("Submit", _("Submit"), css_class="btn btn-default"),
```

Comment:
Ditto. Someone unacquainted with the codebase/crispy forms will have trouble understanding which one of the two submits is the name and which one is the value. Hence it's better to have the `name` (first argument) in all lowercase.